pages/ecosystem.vue

```diff
@@ -402,3 +416,17 @@ const filteredMembersSorted = computed<Member[]>(() => {
   }
 }
 </style>
+
+<style lang="scss">
+@use "~/assets/scss/carbon.scss";
```

Comment:
I think that this import is not necessary.

Reply:
Doesn't this give us the ability to use the `carbon.breakpoint-down(md)`?

Reply:
You're right, I didn't see the breakpoint line.